### PR TITLE
Small changes based on Jorge's feedback

### PR DESCRIPTION
**Pull request by indiesquidge**

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -93,4 +107,6 @@ li { list-style: none; }
   background-color: $site-brown;
 }
 
-#sign-in { margin: 50px; }
+#cart-show {
```
Comment (indiesquidge) [inline app/assets/stylesheets/custom.css.scss]: Stfu, Hound, jesus.

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -16,9 +16,12 @@ body {
   font-family: Avenir;
   color: $site-black;
   background-color: $site-white;
-  padding-top: 70px;
 }
 
+.container { padding-top: 46px; }
+
+.container.nav { padding-top: 0; }
```
Comment (indiesquidge) [inline app/assets/stylesheets/custom.css.scss]: Don't know how to do this. This fucks it up:

```css
.container { padding-top: 46px;
    .nav { padding-top: 0; }
}
```
Hound is bitch about using id selectors, and I can't figure out another way around it. If any of you can then I'm all ears. I'm really getting sick of CSS's shit.

File: app/assets/stylesheets/custom.css.scss [hunk 3]
```diff
@@ -36,6 +39,13 @@ body {
   padding: 5px;
 }
 
+/* navbar dropdown */
```
Comment (indiesquidge) [inline app/assets/stylesheets/custom.css.scss]: Okay, Hound. Makes no sense, but sure.

File: app/assets/stylesheets/custom.css.scss (hunk 1 above)
Comment (paulgrever) [inline app/assets/stylesheets/custom.css.scss]: HAHAH!

**Comment by paulgrever** [comment]
@indiesquidge  I think we should limit the cuss words, as future employers may review these logs. Chances are slim, but possible. But the work here looks good!

**Comment by indiesquidge** [comment]
You bet your ass they will review them!

**Comment by indiesquidge** [comment]
....I'll keep it down :expressionless:

**Pull request merged**